chore: update issue templates (#5)
uninterrupted-tech/serverless-plugin-webform: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
title: "[FEATURE]"
labels: feature
assignees: u11d-michal-miler, u11d-bartlomiej-galezowski

---

> **Is your feature request related to a problem? Please describe.**

Reviewer: I'd change verbs in title to nouns. What do you think?

```suggestion
**Problem description**
```

Author: Yes, that is more readable.

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: Let's be more positive :) 

```suggestion
A clear and concise description of what the problem is. Ex.: The plugin helps me a lot in development but it'd be even more helpful if ... [...]
```

> **Describe the solution you'd like**

Reviewer: ```suggestion
**Expected solution**
```

A clear and concise description of what you want to happen.

> **Describe alternatives you've considered**

Reviewer: ```suggestion
**Alternatives description**
```
